akka-actor/src/main/scala/akka/pattern/RetrySupport.scala

```diff
@@ -30,17 +30,73 @@ trait RetrySupport {
    * private val sendReceiveRetry: HttpRequest => Future[HttpResponse] = (req: HttpRequest) => retry[HttpResponse](
    *   attempt = () => sendAndReceive(req),
    *   attempts = 10,
+   *   delay = 2 seconds)(
+   *    ec = context.system
+   *    scheduler = context.system.scheduler
+   * )
+   * }}}
+   */
+  def retry[T](attempt: () ⇒ Future[T], attempts: Int, delay: FiniteDuration)(implicit ec: ExecutionContext, scheduler: Scheduler): Future[T] =
+    retryF(attempt, attempts, delay, 1)((_, _) ⇒ ())
+
+  /**
+   * Given a function from Unit to Future, returns an internally retrying Future
+   * The first attempt will be made immediately, each subsequent attempt will be made after 'delay' * 'backoff'
```

Comment:
this description is confusing, it doesn't look like it's exponential backoff but that the subsequent attempts will all be 4.seconds (example below)

Reply:
Yes, you are right. I will change it after we decide how the function should look